doc/routinator.1

```diff
@@ -240,6 +240,12 @@ explicit port.
 .IP
 This option allows to disable this filtering.
 
+.TP
+.B --fresh
+Delete and re-initialize all local data storage before starting. This option
```

Comment:
"All", but not including for example TAL files, does that need to be clarified?

Reply:
The term ‘local data storage’ was meant to convey all the data that Routinator itself is creating.

It now occurs to me that ‘cache dir’ (as used by configuration) is incorrect now. Let’s clean that up with a separate PR. (The whole thing should be called the cache and what is now `routinator::cache` should be renamed into something else.